uber/site_sections/preregistration.py

```diff
@@ -113,23 +113,26 @@ def form(self, session, message='', edit_id=None, **params):
         """
         params['id'] = 'None'   # security!
 
+        # Both the Attendee class and Group class have identically named
+        # address fields. In order to distinguish the two sets of address
+        # fields in the params, the Group fields are prefixed with "group_"
+        # when the form is submitted. To prevent instantiating the Group object
+        # with the Attendee's address fields, we must clone the params and
+        # rename all the "group_" fields.
+        group_params = dict(params)
```

Comment:
Whoops, how did I miss that??

I think we do want it moved back down to after we do a group.apply -- there are some pages where there's no group_ prefix for those fields, so we want to apply the regular fields to the group first in case that's the case.